src/components/homepage/featuredWork.js

```diff
@@ -115,39 +121,41 @@ export default function FeaturedWork({ data }) {
 
   return (
     <FeaturedWorkSection>
-      <ContentCenter>
-        <Content>
-          <H2>Featured Work</H2>
-          <P>
-            Where creativity meets accessibility; see
-            examples of how we've transformed our clients' ideas into beautiful
-            and inclusive digital spaces.
-          </P>
-          <Button to="/ourwork">All Our Work &#8702; </Button>
-        </Content>
-        <FeaturedWorkGrid>
-          {randomWorks.map((work, index) => (
-            <WorkCard
-              key={index}
-              image={work.node.data.image.localFiles[0].childImageSharp.fluid}
-              title={work.node.data.title}
+      <FeaturedWorkBox>
+        <ContentCenter>
+          <Content>
+            <H2>Featured Work</H2>
+            <P>
+              <Strong>TO CHANGE</Strong> Where creativity meets accessibility;
+              see examples of how we've transformed our clients' ideas into
+              beautiful and inclusive digital spaces.
+            </P>
+            <Button to="/ourwork">All Our Work > </Button>
```

Comment:
Another nitpick but we'll probably want to keep the bracket as "&#8702;" here

Reply:
Changed the button to use "⇾"